codegen/transformers.go

```diff
@@ -66,3 +66,13 @@ func defaultGoTypeToSchemaType(v reflect.Type) (schema.ValueType, error) {
 		return schema.TypeInvalid, fmt.Errorf("unsupported type: %s", k)
 	}
 }
+
+type ResolverTransformer func(field reflect.StructField, path string) string
+
+func DefaultResolverTransformer(_ reflect.StructField, path string) string {
```

Comment:
Should it also return an `error` to be consistent with all other APIs and not to break compatibility if we need to add it in the future.

Reply:
Also, for the same reason of having max context and no introducing more options and/or breaking changes it should be also `func(field reflect.StructField, parent reflect.StructField) string, error`.

Reply:
I don't think it should have info about parent, as the only thing that matters is type info + path where you can get it.

Reply:
but that path is basically parent.name + child.name ?